src/palakSJun20/ArrayPairSum.java

```diff
@@ -0,0 +1,24 @@
+package palakSJun20;
+
+public class ArrayPairSum {
+	void findArrayPairSum(int[] array) {
+		System.out.println("Integer numbers, whose sum is equal to value:7");
+		for (int i = 0; i < array.length; i++) {
+			if (!(array[i] == '*')) {
```

Comment:
what is meaning of * here?
could please explain.

Reply:
1st time when it enters in if condition  replace the the  same element with * so next time loop iterate it will check if * is there it won't go inside.